### ISSUE TITLE
Show "automated by" on bot account pages

### ISSUE THREAD

**Issue by stefanbohacek** [mastodon/mastodon#28994]
### Pitch

Two years ago, Twitter was [testing a way to associate an automated account with its creator](https://techcrunch.com/2022/02/16/twitter-officially-launches-labels-to-identify-the-good-bots/). Here's an example from [@southpoleviews](https://twitter.com/southpoleviews):

![Screenshot of a portion of a Twitter account profile page showing ](https://github.com/mastodon/mastodon/assets/3909390/95654e73-492f-4ec4-ad6f-1c7f91009b37)

I'd love to see this implemented on Mastodon as well.

### Motivation

We can already [mark an account as automated on Mastodon](https://superuser.com/questions/1439448/what-is-the-effect-of-ticking-this-is-a-bot-account-in-mastodon). Being able to also more visibly show who created each automated account would be a great way to bring more transparency and promote authors of creative bots.

**Comment by ThisIsMissEm** [comment]
Small note on this: we should probably have some form of confirmation flow, to verify that the named account is really responsible for that bot account. This would likely require some work on the activitypub side of things. (we could show it as unverified / verified then.

**Comment by ryliejamesthomas** [comment]
I think this is a good idea. Only qualm is the language, 'automated by' doesn't really make sense, better to use 
'owner' or 'contact' or 'maintainer' or 'run by' or something.

**Comment by smiba** [comment]
Any reason why this can not be done by simply putting this in the bot's bio or profile metadata field? I'm not sure if this feature adds much more imo

**Comment by stefanbohacek** [comment]
@smiba Yes, you could add information about the bot's creator in the profile description or use the metadata fields, but a more standardized approach would enable building additional features in the future.

For example, I would like to make it so that if someone replies to my bot, I'd like to be cc-ed in the message. Or maybe even receive all of the bot's notifications. (I currently manage [a few dozen automated accounts](https://stefans-creative-bots.glitch.me/).)

**Comment by daprice** [comment]
> Any reason why this can not be done by simply putting this in the bot's bio or profile metadata field?

The required verification that @ThisIsMissEm suggested would be another advantage. Verification of a regular profile field currently requires the creator to waste one of their own profile fields linking back to the bot (and is impossible if the creator has more bots than the 4 allowed profile fields)
